README.md

```diff
@@ -1 +1,122 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top">SET UP LINTERS AND CREATE PULL REQUEST</a>
+
+
+<div align="center">
+
+  <img src="images/images.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+
+</div>
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="https://zbahati.github.io/Hello-Microverse/">visit website</a>
+
+> Describe your project in 1 or 2 sentences.
+
```

Comment:
- [ ] As requested by the previous reviewer, these descriptive lines are meant to guide you, you need to remove them after adding the README to your project.